Source/FakeItEasy/Core/DefaultFakeWrapperConfigurer.cs

```diff
@@ -6,21 +6,27 @@ namespace FakeItEasy.Core
     /// Handles configuring of fake objects to delegate all their calls to a wrapped instance.
     /// </summary>
     internal class DefaultFakeWrapperConfigurer
-        : IFakeWrapperConfigurer

```

Comment:
Good point, yeah let's change all occurrences of 'configurer' to 'configurator'.